README.md

```diff
@@ -596,6 +597,78 @@ use `@Email` and `@NotBlank` together, that causes both validations to run even
 enter a value, validating both that they need to enter a value due to `@NotBlank` and because the
 blank value needs to be a validly formatted email address due to `@Email`.
 
+### Required Inputs
+
+As mentioned above in the [Validating Inputs](#validating-inputs) section, the annotations `@NotEmpty`,
+`@NotBlank`, and `@NotNull` are used to make a field required. The library will automatically
+append a red '(required)' to the end of your input labels.
+
+#### Special Required Field Situations
+
+Sometimes you may have a field that is required, but not through an annotation such as those mentioned above.
+The specific scenario where this might be the case include is when A field uses cross validation in 
+the flows-config file and that validation makes it required.
+
+For this situation, we have provided an attribute which can be added to inputs, `required` which is a boolean
+that defaults to false. If set to true, the library will treat the field as required and append the 
+red `(required)`. 
+
```

Comment:
For the above sentence, maybe it could just say "If set to true, the field will be marked required in the user interface, showing that the application requires it."  The sentence, as you have it, implies that the library will treat it as required, which is not the case.